api/apps/api/test/project/projects.fixtures.ts

```diff
@@ -226,15 +247,18 @@ export const getFixtures = async () => {
       await request(app.getHttpServer())
         .get(`/api/v1/projects/${projectId}`)
         .set('Authorization', `Bearer ${notIncludedUserToken}`),
-    WhenPublishingAProject: async (projectId: string) =>
+    WhenPublishingAProject: async (projectId: string, scenarioId: string) =>
       await request(app.getHttpServer())
         .post(`/api/v1/projects/${projectId}/publish`)
         .send({
           name: 'example project',
           description: 'fake description',
+          location: 'fake location',
           company: { name: 'logo', logoDataUrl: blmImageMock },
           creators: [{ displayName: 'fake name', avatarDataUrl: blmImageMock }],
           resources: [{ title: 'fake url', url: 'http://www.example.com' }],
+          config: { baseUrl: 'example/png', cookie: 'randomCookie' },
+          scenarioId,
```

Comment:
in hindsight, I think we should have labelled this `featuredScenarioId` in the DTO for this, but let's not get there now 😓

Reply:
In fact, I think I would change it as you suggested, mainly because once I start the export here, I would also be able to select which scenarios I want for the export and it can get confusing real fast.